sql/moz-fx-data-shared-prod/firefox_desktop_derived/desktop_dau_distribution_id_history_v1/backfill.yaml

```diff
@@ -0,0 +1,9 @@
+2025-02-13:
+  start_date: 2022-12-31
```

Comment:
Is this as far as we need to go back?

Reply:
Yes. There is no data pripr to that in `telemetry.desktop_active_users`